data/json/mapgen/cs_private_park.json

```diff
@@ -5,95 +5,76 @@
     "om_terrain": [ "cs_private_park" ],
     "weight": 1000,
     "object": {
-      "fill_ter": "t_floor",
+      "predecessor_mapgen": "forest_thick",
```

Comment:
Dense forest may not be appropriate for a park since it is way too thick. Maybe just use `forest`?